vite.config.js

```diff
@@ -0,0 +1,23 @@
+import { fileURLToPath, URL } from 'url'
+
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import vueJsx from '@vitejs/plugin-vue-jsx'
```

Comment:
Do we use JSX?

Reply:
I don't think we do, that's just a poor copy paste on my part with default Vite/Vue3 config...